src/ext/ddtrace.h

```diff
@@ -14,6 +14,13 @@ HashTable class_lookup;
 HashTable function_lookup;
 zend_bool log_backtrace;
 zend_function *current_fbc;
+#if PHP_VERSION_ID < 70000
+zval *original_this;
```

Comment:
Looks like there is a family of `originals` now. Would it make sense to refactor all `call_original()` data to a separate a separate struct ? 

It still could be a non pointer field in this struct to simplify stuff. 
I just feel that would be nice to start wrangling some of wild code we've accrued, and separating concerns sounds like a good start.

WDYT ?

Reply:
Great idea! Done. :)